sdk/core/core-client-rest/review/core-client.api.md

```diff
@@ -78,7 +78,7 @@ export interface ErrorResponse {
 
 // @public
 export interface FullOperationResponse extends PipelineResponse {
-    parsedBody?: RequestBodyType;
+    parsedBody?: RequestBodyType | Record<string, any>;
```

Comment:
`RequestBodyType` is defined as follows:
```ts
export type RequestBodyType = NodeJS.ReadableStream | (() => NodeJS.ReadableStream) | ReadableStream<Uint8Array> | (() => ReadableStream<Uint8Array>) | Blob | ArrayBuffer | ArrayBufferView | FormData | string | null;

```
Which doesn't include a JSON object which makes sense because that type represents request payloads about to be sent on the wire. However, for some reason, `parsedBody` in the response is not typed as, you know, an actual parsed JSON object!

Reply:
Addressed in https://github.com/Azure/azure-sdk-for-js/pull/27000/commits/bf8beb704eace9e1c643949ab758ab27efb38dbc